USA/CA/tehama_county/colleges/el_camino/README.md

```diff
@@ -0,0 +1,68 @@
+This readme should give people everything they need to maintain the scraper.
+
+Please stick to [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/) when modifying this readme.  
+
+Note: Ensure your scraper calls the etl.py file. Even if there is nothing in there now, the scraped data will need to be loaded into our database and the etl.py file is what will handle that!
+
+# Source info
+Data is on the last page
+
+## Data refresh rate
+Annually
+
+## Legal
+Include the Terms of Service (or link to them). Is there anything else we should know?
+
+# Fields to collect:
+_Move these fields to the appropriate list below when you submit your scraper._
+
+
+## Fields unobtainable within our legal guidelines:
+
+## Fields not available:
+* _id
+* _state
+* _county
+* CaseNum
+* FirstName
+* MiddleName
+* LastName
+* Suffix
+* DOB
+* Race
+* Sex
+* ArrestDate
+* FilingDate
+* OffenseDate
+* DivisionName
+* CaseStatus
+* DefenseAttorney
+* PublicDefender
+* Judge
+* ChargeCount
+* ChargeStatute
+* ChargeDescription
+* ChargeDisposition
+* ChargeDispositionDate
+* ChargeOffenseDate
+* ChargeCitationNum
+* ChargePlea
+* ChargePleaDate
+* ArrestingOfficer
+* ArrestingOfficerBadgeNumber
+* BookingNum
+* BookingDate
+* WarrantNum
+* BailAmount
+* SearchIncident
+
+## Fields being collected:
```

Comment:
Are you not collecting any fields?

Reply:
It's just annual data, none of those fields are included.